dot/network/message.go

```diff
@@ -388,7 +390,7 @@ func NewAscendingBlockRequests(startNumber, targetNumber uint, requestedData byt
 	numRequests := diff / MaxBlocksInResponse
 	// we should check if the diff is in the maxResponseSize bounds
 	// otherwise we should increase the numRequests by one, take this
-	// example, we want to sync from 0 to 259, the diff is 259
+	// example, we want to sync from 1 to 259, the diff is 259
```

Comment:
would this make the diff 258?

Reply:
since `diff := targetNumber - (startNumber - 1)` I think 259 is right